ensembl/htdocs/info/genome/variation/prediction/protein_function.html

```diff
@@ -31,14 +31,27 @@ <h1 style="margin-top:15px">Ensembl Variation - Protein function predictions</h1
 single amino acid substitutions in the Ensembl human proteome. This enables us to provide 
 predictions for novel mutations for VEP and API users. We were able to compute predictions
 from at least one tool for over 95% of the human proteins in Ensembl. 
+</p>
+
+<p>
 SIFT predictions are also available for cat, chicken, cow, dog, goat, horse, mouse, pig, rat, sheep and zebrafish.
 </p>
 
 <p>
-These tools are developed by external groups and we provide a brief explanation of
-the approach each takes below, and how we run it in Ensembl. For much more detail
-please see the representative papers listed <a href="#references">below</a>, and the 
-relevant publications available on each tool's website. 
+Pathogenicity scores from CADD, REVEL, MutationAssessor and MetaLR are also available for human variants. 
```

Comment:
Add internal links to the corresponding sections (e.g. CADD, REVEL, ...) ?

Reply:
Good idea!